test/classes/phing/tasks/system/MkdirTaskTest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+require_once 'phing/BuildFileTest.php';
```

Comment:
Please remove the `require_once` statement and add a license file comment.

Reply:
Done